build.sbt

```diff
@@ -1,17 +1,19 @@
-import sbt.Keys.{publishTo, resolvers, scalaVersion}
-import SeleniumTestConfig.{SeleniumTest, seleniumTestFilter, unitTestFilter}
+import SeleniumTestConfig.SeleniumTest
+import sbt.Keys.{publishTo, resolvers, scalaVersion, skip, updateOptions}
 import sbtrelease.ReleaseStateTransformations._
+
 import scala.sys.process._
 
-skip in publish := true
+val scalatest = "org.scalatest" %% "scalatest" % "3.2.0"// if the following PR is merged in v3.2.1, remove "-eU" parameter from options above - PR https://github.com/scalatest/scalatest/pull/1842
 
 lazy val integrationTestSettings: Seq[Def.Setting[_]] = Defaults.itSettings ++ Seq(
   scalaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "scala",
   javaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "java",
   resourceDirectory in IntegrationTest := baseDirectory.value / "src" / "test" / "resources",
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-l", "com.gu.test.tags.annotations.IntegrationTest"),
   testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-eU"),
-  testOptions in IntegrationTest += Tests.Argument(TestFrameworks.ScalaTest, "-eU")
+  testOptions in IntegrationTest += Tests.Argument(TestFrameworks.ScalaTest, "-eU"),
+  libraryDependencies += scalatest % "it",
```

Comment:
we can't put scalatest on the "it" classpath unless we add the IntegrationTest config axis to the project, so it should add here instead of commonDeps.